src/mixins/AddSendRequestByMessage.ts

```diff
@@ -181,8 +181,13 @@ export function AddSendRequestByMessage<TBase extends ConnectorTUIBaseConstructo
         },
         {
           message: "Who is the owner of the RelationshipAttribute?",
-          type: "text",
+          type: "select",
           name: "owner",
+          choices: [
+            { title: "You (Connector)", value: "" },
```

Comment:
An empty string for the `owner` does not mean that you are the `owner`. Instead, the `owner` is `"The other Side or Third Party"`. Therefore, I would also offer the empty string option as the last option.